scripts/sct_process_segmentation.py

```diff
@@ -205,14 +209,22 @@ def main(args):
         vert_lev = ''
     if '-r' in arguments:
         remove_temp_files = arguments['-r']
-    if '-size' in arguments:
-        smoothing_param = arguments['-size']
+    # if '-size' in arguments:
```

Comment:
I think it's always better to remove comments when possible, the history is saved in git anyways.